src/main/java/org/teamresistance/frc/UltraSensor.java

```diff
@@ -0,0 +1,35 @@
+package org.teamresistance.frc;
+
+import org.strongback.command.Command;
+
+import edu.wpi.first.wpilibj.Ultrasonic;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ * @author Tarik C. Brown
+ */
+public class UltraSensor extends Command {
+  public Ultrasonic ping1;
+  public Ultrasonic ping2;
+  public double range;
+  public double range2;
+  public UltraSensor() {
+    // Should be Ultrasonic(Ping Channel, Echo Channel
```

Comment:
This comment about the constructor may not be necessary since `Ctrl+P` in the IDE gives us the same information, and comments like this can go out of date.